### DISCUSSION TITLE
Pylance support for Quarto .qmd file

### DISCUSSION THREAD

**Discussion by ntluong95** [microsoft/pylance-release#6093] [Enhancement]
Currently, there is no support for `.qmd` file from Quarto extension in Vscode.
Since this file is more and more popular, having support for it will benefit the Python community, as well as R community starting learn Python.  For example, pandas and pd does not recognize as module in the attached image below:

![image](https://github.com/microsoft/pylance-release/assets/63569637/e69d9487-4f28-4e60-b615-27bb523ce2f9)

**Comment by debonte** [comment]
Moving this issue to discussions as an enhancement request for comments and upvotes.
